contractions/data/slang_dict.json

```diff
@@ -1,17 +1,46 @@
 {
-    "'aight": "alright",
-    "dunno": "do not know",
-    "howdy": "how do you do",
-    "ima": "I am going to",
-    "innit": "is it not",
-    "iunno": "I do not know",
-    "g'day": "good day",
-    "gonna": "going to",
-    "gotta": "got to",
-    "wanna": "want to",
-    "woulda": "would have",
-    "gimme": "give me",
-    "asap": "as soon as possible",
-    "u": "you",
-    "r ": "are "
+     "'aight": "alright",
+     "abt": "about",
+     "acct": "account",
+     "altho": "although",
+     "asap": "as soon as possible",
+     "avg": "average",
+     "b4": "before",
+     "bc": "because",
+     "btw": "by the way",
+     "chk": "check",
+     "convo": "conversation",
+     "cya": "see ya",
+     "diff": "different",
+     "dunno": "do not know",
+     "g'day": "good day",
+     "gimme": "give me",
+     "gonna": "going to",
+     "gotta": "got to",
+     "howdy": "how do you do",
+     "ima": "I am going to",
+     "innit": "is it not",
+     "iunno": "I do not know",
+     "kk": "okay ",
```

Comment:
I think we should not need the space at the end here in the translated result?

Reply:
You are right - no space is needed! That is my mistake.